distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Replace_Helpers.enso

```diff
@@ -0,0 +1,53 @@
+from Standard.Base import all
+
+import Standard.Base.Errors.Empty_Error.Empty_Error
+import Standard.Base.Errors.Illegal_Argument.Illegal_Argument
+
+import Standard.Database.Data.Table.Table as Database_Table
+
+import project.Data.Table.Table
+import project.Data.Set_Mode.Set_Mode
+from project.Errors import Missing_Input_Columns, No_Such_Column, Non_Unique_Key, Unmatched_Rows_In_Lookup
+
+replace : Table -> Table | Map -> (Text | Integer) -> (Text | Integer | Nothing) -> (Text | Integer | Nothing) -> Boolean -> Problem_Behavior -> Table ! Missing_Input_Columns | Non_Unique_Key | Unmatched_Rows_In_Lookup
+replace base_table:(Table | Database_Table) lookup_table:(Table | Database_Table | Map) column:(Text | Integer) from_column:(Text | Integer | Nothing)=Nothing to_column:(Text | Integer | Nothing)=Nothing allow_unmatched_rows:Boolean=True on_problems:Problem_Behavior=Problem_Behavior.Report_Warning =
+    case lookup_table of
+        _ : Map ->
+            if from_column.is_nothing.not || to_column.is_nothing.not then Error.throw (Illegal_Argument.Error "If a Map is provided as the lookup_table, then from_column and to_column should not also be specified.") else 
+                handle_empty_lookup_table ~action =
+                    if lookup_table.is_empty.not then action else
+                        ## If the lookup table is empty but the base table is
+                           not, and allow_unmatched_rows=False, throw
+                           `Unmatched_Rows_In_Lookup` with the first row of the
+                           table. Otherwise, attach a warning.
+                        if base_table.row_count == 0 || allow_unmatched_rows then Warning.attach (Empty_Error.Error lookup_table) base_table else
```

Comment:
```suggestion
                        if base_table.row_count == 0 || allow_unmatched_rows then Warning.attach (Empty_Error.Error "lookup_table") base_table else
```

Without the quotes the message will contain the contents of the map. Which being empty won't be so informative for  the user.